doc/library/config.txt

```diff
@@ -452,31 +456,38 @@ import theano and print the config variable, as in:
         automatically to get more memory. But this can cause
         fragmentation, see note above.
 
-.. attribute:: config.gpuarray.preallocate
+.. attribute:: config.lib.cnmem
 
-    Float value
+    .. note::
 
-    Default: 0
+        This value allocates GPU memory ONLY when using (:ref:`cuda`)
+        and has no effect when the GPU backend is (:ref:`gpuarray`).
+        For the new backend, please see ``config.gpuarray.preallocate`` 
 
-    Controls the preallocation of memory with the gpuarray backend.
+    Float value: >= 0
 
-    The value represents the start size (either in MB or the fraction
-    of total GPU memory) of the memory pool. If more memory is needed,
-    Theano will try to obtain more, but this can cause memory
-    fragmentation.
+    Controls the use of `CNMeM <https://github.com/NVIDIA/cnmem>`_ (a
+    faster CUDA memory allocator). In Theano dev version until 0.8
+    is released.
```

Comment:
Instead of the last sentence:
Applies to the old GPU backend :ref:`cuda` up to Theano release 0.8.